launch-game.sh

```diff
@@ -25,7 +25,17 @@ mono OpenRA.Game.exe Engine.LaunchPath="$MODLAUNCHER" $MODARG "$@"
 
 # Show a crash dialog if something went wrong
 if [ $? != 0 ] && [ $? != 1 ]; then
-	ERROR_MESSAGE="OpenRA has encountered a fatal error.\nPlease refer to the crash logs and FAQ for more information.\n\nLog files are located in ~/.openra/Logs\nThe FAQ is available at http://wiki.openra.net/FAQ"
+	if [[ "$OSTYPE" == "darwin"* ]]; then
+		LOGS="~/Library/Application Support/OpenRA/Logs/"
+	else
+		if [ ! -d "$XDG_CONFIG_HOME" ]; then
+			LOGS="~/.config/openra/Logs"
+		else
+			LOGS="$XDG_CONFIG_HOME/openra/Logs"
+		fi
```

Comment:
Aren't we missing the legacy path here? I'd have written it this way:

```suggestion
LOGS="${XDG_CONFIG_HOME:-${HOME}/.config}/openra/Logs"

if [[ ! -d "${LOGS}" && -d "${HOME}/.openra/Logs" ]]; then
	LOGS="${HOME}/.openra/Logs
fi
```

That matches the code in Platform.cs, too.